core-services/prow/02_config/_plugins.yaml

```diff
@@ -1903,13 +1904,14 @@ bugzilla:
       target_release: 4.2.z
       validate_by_default: true
       dependent_bug_target_release: 4.3.0
-    # TODO: turn this on when we branch
-    # openshift-4.3:
-    #   target_release: 4.3.z
-    #   validate_by_default: true
+    openshift-4.3:
+      target_release: 4.3.z
+      validate_by_default: true
+      dependent_bug_target_release: 4.4.0
     release-4.3:
-      target_release: 4.3.0
+      target_release: 4.3.z
```

Comment:
We don't want this until we cut 4.3.0, right?

Reply:
we wouldn't want people merging to release-4.3 at all at this point, they should be using master, so it'd be a good thing...